src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -64,8 +64,8 @@ public function __construct(array $defaultOptions = [], int $maxHostConnections
             'dnsCache' => [[], [], []],
         ];
 
-        // Skip configuring HTTP/2 push when it's unsupported or buggy, see https://bugs.php.net/76675
-        if (\PHP_VERSION_ID < 70215 || \PHP_VERSION_ID === 70300 || \PHP_VERSION_ID === 70301) {
+        // Skip configuring HTTP/2 push when it's unsupported or buggy, see https://bugs.php.net/76675 and https://bugs.php.net/bug.php?id=77535
+        if (\PHP_VERSION_ID < 70217 || (\PHP_VERSION_ID >= 70300 && \PHP_VERSION_ID <= 70303)) {
```

Comment:
I suggest using `< 70304` instead, which would involve the version containing the fix rather than the previous one (and would be consistent with what we do elsewhere.